### PR TITLE
(maint) - Update PDK Install

### PR DESCRIPTION
**Pull request by jordanbreen28**
## Summary
Provide a detailed description of all the changes present in this pull request.

## Additional Context
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process behind the implementation.

## Related Issues (if any)
Mention any related issues or pull requests.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified.

### CODE REVIEW

**Review by GSPatton — APPROVED** [review]

**Comment by GSPatton** [comment]
do not merge. Waiting for updates to Bolt

**Comment by jordanbreen28** [comment]
Can be seen working [here](https://github.com/puppetlabs/Puppet.Dsc/actions/runs/5966668432/job/16186820919)

**Review by pmcmaw — APPROVED** [review]

File: .github/workflows/puppetize.yml [hunk 1]
```diff
@@ -87,13 +87,15 @@ jobs:
         env:
           FORGE_TOKEN: ${{ secrets.FORGE_API_TOKEN }}
         run: |
+          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
+          refreshenv
           Import-Module -Name PSDesiredStateConfiguration -Force
-          Import-Module -Name Puppet.Dsc -Force
+          Import-Module -Name ./src/Puppet.Dsc -Force
```
Comment (GSPatton) [inline .github/workflows/puppetize.yml:93]: one thing... should we revert this back to not use the local version of the module?

Comment (jordanbreen28) [reply .github/workflows/puppetize.yml:93]: going to wait until its released to change it back incase we need to release any dsc modules

**Pull request merged**